### PR TITLE
Move nobanner and fileplatform from Themes to Resources

### PR DESCRIPTION
**Pull request by JosJuice**
fileplatform is moved so it's in the same place as the other platform icons, and nobanner is moved just because it fits better in Resources. Both of them were identical in all of Dolphin's themes.

### CODE REVIEW

File: Source/Core/DolphinQt2/Resources.cpp [hunk 1]
```diff
@@ -56,10 +57,9 @@ void Resources::Init()
 	for (QString rating : ratings)
 		m_ratings.append(QPixmap(rating.prepend(sys_dir)));
 
-	QString theme_dir = QString::fromStdString(File::GetThemeDir(SConfig::GetInstance().theme_name));
-	m_misc.append(QPixmap(QStringLiteral("nobanner.png").prepend(theme_dir)));
-	m_misc.append(QPixmap(QStringLiteral("dolphin_logo.png").prepend(theme_dir)));
-	m_misc.append(QPixmap(QStringLiteral("Dolphin.png").prepend(theme_dir)));
```
Comment (JosJuice) [inline Source/Core/DolphinQt2/Resources.cpp]: Using `theme_dir` with `dolphin_logo` and `Dolphin` seems like a mistake. The files are only in the Resources directory. Is that correct? (cc @spxtr)

**Comment by phire** [comment]
I kind of think someone might want to theme those resources, along with the platform icons. 
But since we don't really have a theming community anyway, this doesn't seem like a bad idea.

**Comment by JosJuice** [comment]
A possible long-term plan is to let themes override resources, so that they only need to provide platform icons if they really do want custom ones.

**Comment by phire** [comment]
Ah ok. That makes sense. 
Perhaps put your future intentions in the PR description in the future. 

LGTM

**Pull request merged**